src/sdkconfig

```diff
@@ -0,0 +1,962 @@
+#
```

Comment:
is this file necessary since it gets autogenerated?

Reply:
It needs to be generated at the first run and wont be regenerated later